parameters.sh

```diff
@@ -0,0 +1,3 @@
+# Number of cores
```

Comment:
I would rename the file `parameters` or `parameters.conf`

Reply:
done